README.md

```diff
@@ -84,7 +84,7 @@ end
 
 Command match data includes `match['bot']`, `match['command']` and `match['expression']`. The `bot` match always checks against the `SlackRubyBot::Config.user` and `SlackRubyBot::Config.user_id` values obtained when the bot starts.
 
-The `command` method can take strings, which will have be escaped with `Regexp.escape`, and regular expressions.
+The `command` method can take strings, which will have to escaped with `Regexp.escape`, and regular expressions.
```

Comment:
to _be_ escaped I think is more correct ;)

Reply:
yes that make sense :)